Bugfix_If_I_speak_Heading

kimberleyworld opened the pull request:
- Added a margin to the bottom of the H4 element except when at small screen sizes. 
- Applied new formatting standards. 
- Updated staging site but, I think this might  be because the staging site is slightly behind., the **If I speak** category doesn't appear in the footer. When I navigate there directly using the URL there are no episodes to list. I have messaged Pietro about this so will update the thread if I get a clearer understand. 

https://app.asana.com/0/1207702320374002/1208867451959630/f

kimberleyworld committed:
reformating catagory-if-i-speak.php

kimberleyworld committed:
Added a margin-bottom to all screen sizes bigger than small

varffs reviewed (changes requested):
Only procedural things to look that. I know it is very minor but a note in the changelog and cleaner formatting of the pull request name are needed. Feel free to merge yourself once you've done those 2 things.

kimberleyworld committed:
Update CHANGELOG.md

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 6c3c337 according to the output
from Prettier.

Details: https://github.com/novaramedia/novaramedia-com/pull/319
